### ISSUE TITLE
Facing Issues?

### ISSUE THREAD

**Issue by hashirshoaeb** [justiceserv/gh-page#1]
Hey!

I'm author of [hashirshoaeb/home](https://github.com/hashirshoaeb/home). Let me know if you face any issue, while using my code. :) 

Best,
Hashir Shoaib

**Comment by justiceserv** [comment]
Hi! I was getting issues while deploying right into my forked repo (Which i deleted now) using Github Action, which was 
[
name: Node.js CI

on:
  push:
    branches: [ master ]
  pull_request:
    branches: [ master ]

jobs:
  build:

    runs-on: ubuntu-latest

    strategy:
      matrix:
        node-version: [10.x, 12.x, 14.x]

    steps:
    - uses: actions/checkout@v2
    - name: Use Node.js ${{ matrix.node-version }}
      uses: actions/setup-node@v1
      with:
        node-version: ${{ matrix.node-version }}
    - run: npm install
    - run: npm run build
    - run: npm run deploy](url)
I wasn't able to and deploy in / (not /home), is there anyway i can fixed that so I can put this in my forked repo?

**Comment by hashirshoaeb** [comment]
Your are saying that, you want to deploy it in (justiceserv.github.io) but not it (justiceserv.github.io/home). And also you want to keep the react and deployed code in the forked repo?  Am i right?

**Comment by justiceserv** [comment]
yes!

**Comment by hashirshoaeb** [comment]
Well, I don't know how to use GitHub actions, but I can tell you the other way. 

Steps:
- Fork the [hashirshoaeb/home](https://github.com/hashirshoaeb/home)
- Name it justiceserv.github.io
- Read this link https://help.github.com/en/github/working-with-github-pages/about-github-pages
  - Key points to know that.
  - The code in repo: **justiceserv.github.io** branch: **master** will be publish at **justiceserv.github.io**
  - The code in repo: **home** branch: **gh-pages** will be publish at **justiceserv.github.io/home**
- What you can do is, keep the react code in some other branch. Let's say react-branch and publish the build folder in master branch. You can use the script that I wrote for this purpose. It's in [pages.js](https://github.com/hashirshoaeb/home/blob/master/pages.js)

**Comment by hashirshoaeb** [comment]
What i'm using is, 
 I have repo: [hashirshoaeb/home](https://github.com/hashirshoaeb/home) for react code and repo: [hashirshoaeb.github.io](https://github.com/hashirshoaeb/hashirshoaeb.github.io) for deplovment. 

For deployment:
 I first build the project using
```bash
npm run build
``` 
then for deployment
```bash
npm run custom-deploy
```
custom-deploy actually runs
```bash
node pages.js
```
and pages.js publishs the build folder at repo: **hashirshoaeb.github.io** branch: **master**

**Comment by hashirshoaeb** [comment]
This can be automated using GitHub actions, but i don't know how to do it.

**Comment by justiceserv** [comment]
oh i'll try it right now! By the way, can my portfolio added into your readme in master branch after doing them?

**Comment by hashirshoaeb** [comment]
Yes Sure, I'll be happy to do that. 💯 Just ping me when you are done with deployment.
